internal/printer/printer_test.go

```diff
@@ -45,8 +47,9 @@ type want struct {
 	styles         []string
 	scripts        []string
 	getStaticPaths string
+	code           string
+	skipHoist      bool // HACK: sometimes `getStaticPaths()` appears in a slightly-different location. Only use this if needed!
```

Comment:
So if there’s nothing between imports and exports, nothing will get hoisted because nothing needs to be. Unfortunately in our tests that means that `Astro` global and `$$metadata` can appear on either side, so this was my quick hack just for the test I added.

Reply:
To be clear: it’s probably not a bug; just that hoisting can affect the print-to-js order which affects our tests